examples/scan.py

```diff
@@ -0,0 +1,82 @@
+import os
+
+import httpx
+from rich import print
+
+import acuvity
+from acuvity import Acuvity, Guard, GuardName
+
+s = Acuvity(
+    # not required at all if set in environment variables
+    security=acuvity.Security(
+       token=os.getenv("ACUVITY_TOKEN", ""),
+    )
+)
+
+# list guards
+print("\n\n ****** AVAILABLE GUARDS ******\n\n")
+print(s.apex.list_available_guards())
+
+
+guard_config = {
+        "guardrails": [
+            {
+                "name": "prompt_injection",
+                "threshold": ">= 0.2"
+            },
+            {
+                "name": "jail_break",
+                "threshold": ">= 0.7"
+            },
+            {
+                "name": "malicious_url",
+                "threshold": ">= 0.7"
+            },
+            {
+            "name": "pii_detector",
+            "count_threshold": 2,
+            "matches": {
+                "email_address": {
+                    "threshold": "> 0.2",
+                    "count_threshold": 1
+                    },
+                "person": {
+                    "threshold": ">= 1.0"
+                    },
+                "money_amount": {
+                    "threshold": ">= 0.5"
+                    }
+                }
+            },
+            {
+                "name": "secrets_detector"
+            },
+            {
+                "name": "bias"
+            },
+            {
+                "name": "toxicity"
+            },
+            {
+                "name": "language"
+            }
+        ]
+    }
+
+text1 = "croporate sales number are 10k filling, in.abcd@gmail.com, 123abcd@yahoo.com fuck you"
```

Comment:
typo: `croporate`

I understand we want to use `fuck you` for toxicity or nsfw. but is there anything which sounds a bit friendlier 😅